index.php

```diff
@@ -107,4 +107,4 @@ function renderDifferentPage($id)
   </div>
 </body>
 
-</html>
\ No newline at end of file
+</html>
```

Comment:
Řádek 12. - zbytečně dotazuješ všechny sloupce v tabulce users, upravil bych dotaz takto (vyžadoval jen fullname, který je použit na ř. 50):
`$sql = "SELECT fullname FROM users WHERE id = ?";`